### PR TITLE
🍰 Hide Pagenation On Short Transactionlist

### LINKED ISSUE #1804: 🐛 [Bug] Hide Pagination On Empty Transaction List
## 🐛 Bugreport

If there are no transactions, please disconnect pagenation. 

FireShot Capture 1117 -  - localhost

All paginations including the admin have to be checked.

### PR DESCRIPTION
**Pull request by Tirokk**
## 🍰 Pullrequest

Hide pagenation on short transactionlist.

### Issues

- fixes #1804

### Todo

- [x] for seeding give bob@baumeister.de 28 transactions to show pagination on transaction list
- [x] Gradido account sum does not fit to the sum of the seeded transaction: see https://github.com/gradido/gradido/pull/1875#issuecomment-1118608209
  - we have a quick fix for that, see https://github.com/gradido/gradido/pull/1875/commits/c10115397568550cfaf69eb5317a97e2f8e0f049
- [x] add tests
- [x] clean up: search `Wolle`

### CODE REVIEW

**Comment by Tirokk** [comment]
### Problem

GDD sum does not fit to the transactions in the list.

Bildschirmfoto 2022-05-05 um 13 55 20

Bildschirmfoto 2022-05-05 um 13 56 14

File: .gitignore [hunk 1]
```diff
@@ -11,4 +11,9 @@ package-lock.json
 /deployment/bare_metal/nginx/update-page/updating.html
 /deployment/bare_metal/log
 /deployment/bare_metal/backup
-/.nvmrc
+
+# Node Version Manager configuration file
+.nvmrc
+
+# Apple macOS folder attribute file
+.DS_Store
```
Comment (ogerly) [inline .gitignore:19]: this is part of the PR #1902

File: backend/src/seeds/creation/index.ts [hunk 2]
```diff
@@ -1,23 +1,143 @@
...
+    amount: 74.1,
+    memo: 'von Heidi',
+  },
+  {
+    amount: 4.5,
+    memo: 'von Peter',
+  },
+  {
+    amount: 5.8,
+    memo: 'von Frau Rottenmeier',
```
Comment (ogerly) [inline backend/src/seeds/creation/index.ts:115]:
```suggestion
    memo: 'von Fräulein Rottenmeier',
```

Comment (ogerly) [reply backend/src/seeds/creation/index.ts:115]: Fräulein Rottenmeier

(Das Fräulein Rottenmeier ist die Haushälterin der Familie Sesemann. )

File: admin/.gitignore [hunk 3]
```diff
@@ -10,4 +10,3 @@ coverage/
 
 # emacs
 *~
```
Comment (ogerly) [inline admin/.gitignore:12]: this is part of the PR #1902

File: backend/.gitignore [hunk 4]
```diff
@@ -6,4 +6,3 @@ package-json.lock
 coverage
 # emacs
 *~
```
Comment (ogerly) [inline backend/.gitignore:8]: this is part of the PR #1902

File: database/.gitignore [hunk 5]
```diff
@@ -25,4 +25,3 @@ package-lock.json
 coverage/
 
 *~
```
Comment (ogerly) [inline database/.gitignore:27]: this is part of the PR #1902

File: backend/src/seeds/creation/index.ts (hunk 2 above)
Comment (ogerly) [inline backend/src/seeds/creation/index.ts:115]:
```suggestion
    memo: 'von Fräulein Rottenmeier',
```

**Review by ogerly — CHANGES_REQUESTED** [review]

File: backend/src/seeds/creation/index.ts [hunk 6]
```diff
@@ -1,23 +1,143 @@
...
+  },
+]
+let bobsSum = 0
```
Comment (Mogge) [inline backend/src/seeds/creation/index.ts:118]: Not needed. This was debugging

File: backend/src/seeds/creation/index.ts [hunk 7]
```diff
@@ -1,23 +1,143 @@
...
+const bobsTransactions: CreationInterface[] = []
+bobsSendings.forEach((sending) => {
+  bobsSum = bobsSum + sending.amount
```
Comment (Mogge) [inline backend/src/seeds/creation/index.ts:121]: Not needed

File: backend/src/seeds/creation/index.ts [hunk 8]
```diff
@@ -1,23 +1,143 @@
...
-    amount: 1000,
+    amount: 10,
     memo: 'Herzlich Willkommen bei Gradido!',
-    creationDate: nMonthsBefore(new Date()),
-    confirmed: true,
-    moveCreationDate: 12,
   },
   {
+    amount: 10,
+    memo: 'von Betty',
```
Comment (Mogge) [inline backend/src/seeds/creation/index.ts:11]: We seed creations, which are not transactions. So all this memos are missleading

File: backend/src/seeds/index.ts [hunk 9]
```diff
@@ -66,7 +66,10 @@ const run = async () => {
 
   // create GDD
   for (let i = 0; i < creations.length; i++) {
+    const now = new Date().getTime() // we have to wait a little! quick fix for account sum problem of bob@baumeister.de
```
Comment (Mogge) [inline backend/src/seeds/index.ts:69]:
```suggestion
    const now = new Date().getTime() // we have to wait a little! quick fix for account sum problem of bob@baumeister.de (see https://github.com/gradido/gradido/issues/1886)
```

**Review by Mogge — CHANGES_REQUESTED** [review]

File: backend/src/seeds/creation/index.ts (hunk 2 above)
Comment (Tirokk) [reply backend/src/seeds/creation/index.ts:115]: It's a fauxpas and @ogerly is right.
I think we shloud correct this! Or? @Mogge

Comment (Tirokk) [reply backend/src/seeds/creation/index.ts:115]: It's a fauxpas and @ogerly is right.
I think we shloud correct this! Or? @Mogge

Comment (Tirokk) [reply backend/src/seeds/creation/index.ts:115]: Done

File: backend/src/seeds/creation/index.ts (hunk 6 above)
Comment (Tirokk) [reply backend/src/seeds/creation/index.ts:118]: same same 👇🏼

File: backend/src/seeds/creation/index.ts (hunk 7 above)
Comment (Tirokk) [reply backend/src/seeds/creation/index.ts:121]: Ups!
Overlooked it at cleanup.
Thanks for your close look @Mogge

File: backend/src/seeds/creation/index.ts (hunk 8 above)
Comment (Tirokk) [reply backend/src/seeds/creation/index.ts:11]: I don't understand exactly … @Mogge 
What shall I do in your opinion?

File: backend/src/seeds/index.ts (hunk 9 above)
Comment (Tirokk) [reply backend/src/seeds/index.ts:69]: that's a good hint @Mogge !

Done

File: backend/src/seeds/creation/index.ts (hunk 8 above)
Comment (Tirokk) [reply backend/src/seeds/creation/index.ts:11]: Thanks for talking! Now I got it …

**Review by ogerly — APPROVED** [review]

**Review by Mogge — APPROVED** [review]

**Pull request merged**